### PR TITLE
update changelog for 3.1.0 release

### PR DESCRIPTION
**Pull request by danrosenthal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 3.1.0 - 2018-11-16
+
+### Enhancements
+
+- Allowed `Avatar` to fall back to `initials` when the image fails to load ([#557](https://github.com/Shopify/polaris-react/pull/557))
```
Comment (solonaarmstrong) [inline CHANGELOG.md]: _Technically_ permissive language is discouraged, but changing the tense here was a good call.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Added `onScrolledToBottom` prop to `Scrollable` ([#568](https://github.com/Shopify/polaris-react/pull/568))
+
+### Bug fixes
+
+- Fixed `Action`s selector in `Page`'s `Header` component ([#2487](https://github.com/Shopify/polaris-react/pull/2487))
```
Comment (AndrewMusgrave) [inline CHANGELOG.md]: Invalid PR number

Comment (solonaarmstrong) [inline CHANGELOG.md]: `Action`'s ?

**Comment by danrosenthal** [comment]
@solonaarmstrong could you take one more look?

**Pull request merged**